system/lib/libc/musl/arch/emscripten/bits/syscall.h

```diff
@@ -573,7 +573,7 @@
 #define SYS_getdents64		220
 #define SYS_fcntl64		221
 /* 223 is unused */
-#define SYS_gettid		224
+/*#define SYS_gettid		224*/
```

Comment:
these changes don't look related to libcxxabi? are they are workaround for something?

Reply:
Oh yeah sorry, there is code in libcxxabi the does `if defined(SYS_gettid)`.     I guess maybe we should look at whether we need any of these defines.  Certainly these two are not supposed syscalls.

https://github.com/llvm-mirror/libcxxabi/blob/master/src/cxa_guard_impl.h#L121

Do we even need any of these macros?

Reply:
I think they are used for musl syscalls? (the ones we actually use)

Sounds good to remove ones we don't need. How about doing it like the existing `/* X is unsed */`?

Reply:
I'm going to work on followup change to cleanup this file and remove any syscalls we don't implement.

Reply:
This change is no longer part of this CL.